tests/Integration/ParserTest.php

```diff
@@ -91,4 +91,34 @@ public function testIssue267()
         $this->assertEquals(Image::class, \get_class($document->getObjectById('128_0')));
         $this->assertStringContainsString('4 von 4', $document->getText());
     }
+
+    /**
+     * Test that issue related pdf can now be parsed
+     *
+     * @see https://github.com/smalot/pdfparser/issues/322
+     */
+    public function testIssue322()
+    {
+        $filename = $this->rootDir.'/samples/bugs/Issue322.pdf';
+
+        $document = $this->fixture->parseFile($filename);
+
+        $this->assertStringContainsString('this text isn’t working properly, I’ve edited it in Google Documents', $document->getText());
+    }
+
+    /**
+     * Test that issue related pdf can now be parsed
+     * License of the content taken from https://stackoverflow.com in the sample PDF:
+     * CC BY-SA 2.5 https://creativecommons.org/licenses/by-sa/2.5/
+     *
+     * @see https://github.com/smalot/pdfparser/issues/334
+     */
+    public function testIssue334()
+    {
+        $filename = $this->rootDir.'/samples/bugs/Issue334.pdf';
```

Comment:
That file contains screenshots from questions and answers out of Stackoverflow. I am not sure if you can include them in that state. Is there a way to make the PDF as small as possible and/or remove usernames and other copyright-related entities (user images). I am not an attorney, but I am afraid that might infringe copyright law.

Reply:
You're probably right! The CC license only covers the content of the questions and answers. I'll try to reduce the sample PDF to the text.

Reply:
Oh, by the way...
> remove usernames

that would explicitly go against the attribution part of the CC license... 😉 But I'll try to strip off any images without losing the parts containing the issue (fingers crossed)

Reply:
I was not talking about the PDF of #319, but #334!

Reply:
Maybe I got you wrong, but well, we talked about both of them really, but inside this review conversation, it's all about #334. And the CC license applies to that PDF.

_(Sorry for resolving this, I'm not too familiar with all the processes of OS collaboration, and I thought that regarding #334, everything was now ok, as I reduced the sample via the additional commit)_

Reply:
Based on the following text from Stackoverflow (Soure: https://stackoverflow.com/help/licensing):

```
As noted in the Stack Exchange Terms of Service and in the footer of every page, all publicly 
accessible user contributions are licensed under Creative Commons Attribution-ShareAlike 
license as follows:

* Content contributed before 2011-04-08 (UTC) is distributed under the terms of CC BY-SA 2.5.

* Content contributed from 2011-04-08 up to but not including 2018-05-02 (UTC) is distributed under 
the terms of CC BY-SA 3.0.

* Content contributed on or after 2018-05-02 (UTC) is distributed under the terms of CC BY-SA 4.0.
The license applicable for each Question and Answer revision is available on the post timeline. 
See this post for more information.

Please read the terms of service and the full legal text of the license carefully for more details on 
how your content can be used and for how you can use publicly accessible content contributed to 
the site by other users.
```
The content of that file we talked about falls under the CC [BY-SA 2.5](https://creativecommons.org/licenses/by-sa/2.5/) (its older than 2011-04-08).

CC BY-SA allows us to "remix, transform, and build upon the material for any purpose, even commercially.", therefore removals of usernames are allowed. Anyway, I did some research and found that usernames are not "creative work", therefore not subject for copyright law.

So file is OK and subject solved. Thank you for helping out here.